alembic/versions/afc2f95a298b_remove_gccrashes.py

```diff
@@ -0,0 +1,139 @@
+"""remove gccrashes
+
+Revision ID: afc2f95a298b
+Revises: 77395783fce5
+Create Date: 2017-08-29 14:21:59.825946
+
+"""
+
+from alembic import op
+from sqlalchemy import Column
+from sqlalchemy.dialects.postgresql import INTEGER, NUMERIC, REAL, TIMESTAMP
+
+from socorro.lib.migrations import load_stored_proc
+
+
+# revision identifiers, used by Alembic.
+revision = 'afc2f95a298b'
+down_revision = '77395783fce5'
+
+
+def upgrade():
+    op.execute(
+        'DROP FUNCTION update_gccrashes(date, boolean, interval)'
+    )
+    op.execute(
+        'DROP FUNCTION backfill_gccrashes(date, interval)'
+    )
+    op.drop_table('gccrashes')
```

Comment:
Like the other PR, I think we should drop the table and the functions with sql and an IF EXISTS part.

Reply:
will do.